diff --git a/CMakeLists.txt b/CMakeLists.txt
index c38243c..3925f05 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,8 +1,12 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.16)
 
 # https://cmake.org/cmake/help/latest/command/project.html
 project(box2d VERSION 2.4.1)
 
+if(MSVC)
+    set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+endif()
+
 # set(CMAKE_CONFIGURATION_TYPES "Debug;RelWithDebInfo" CACHE STRING "" FORCE)
 
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
